ivy/functional/backends/paddle/experimental/activations.py

```diff
@@ -47,28 +47,40 @@ def thresholded_relu(
     )
 
 
-def relu6(x: paddle.Tensor, /, *, out: Optional[paddle.Tensor] = None) -> paddle.Tensor:
+@with_unsupported_device_and_dtypes(
+    {"2.5.1 and below": {"cpu": ("bfloat16",)}}, backend_version
+)
+def relu6(
+    x: paddle.Tensor, /, *, out: Optional[paddle.Tensor] = None, complex_mode="jax"
+) -> paddle.Tensor:
     if x.dtype in [paddle.float32, paddle.float64]:
         return F.relu6(x)
     if paddle.is_complex(x):
         return paddle.complex(F.relu6(x.real()), F.relu6(x.imag()))
     return F.relu6(x.cast("float32")).cast(x.dtype)
 
 
+@with_unsupported_device_and_dtypes(
+    {"2.5.1 and below": {"cpu": ("bfloat16",)}}, backend_version
+)
 def logsigmoid(
-    input: paddle.Tensor, /, *, out: Optional[paddle.Tensor] = None
+    input: paddle.Tensor, /, *, out: Optional[paddle.Tensor] = None, complex_mode="jax"
 ) -> paddle.Tensor:
     if input.dtype in [paddle.float32, paddle.float64]:
         return F.log_sigmoid(input)
     if paddle.is_complex(input):
         return paddle_backend.log(
             paddle_backend.divide(
-                1.0, (paddle_backend.add(1.0, paddle_backend.exp(input)))
+                1.0, (paddle_backend.add(1.0, paddle_backend.exp(-input)))
             )
         )
     return F.log_sigmoid(input.cast("float32")).cast(input.dtype)
 
 
+@with_unsupported_device_and_dtypes(
+    {"2.5.1 and below": {"cpu": ("bfloat16", "complex64", "complex128")}},
+    backend_version,
+)
```

Comment:
Since the paddle backend's `elu` should be able to support complex numbers, so should its `selu` (using the `jax_like` function)
```suggestion
@with_unsupported_device_and_dtypes(
    {"2.5.1 and below": {"cpu": ("bfloat16",)}},
    backend_version,
)
```